### ISSUE TITLE
Vim mode: Cmd enter jumps to first block instead of toggling

### ISSUE THREAD

**Issue by gvteja** [roam-unofficial/roam-toolkit#137]
I believe Roam recently changed their cmd+enter behavior to focus on the first block if the focus is not already on a block. Since vim mode's focus is different from Roam, pressing cmd+enter is going to the first block instead of toggling the todo in the selected (as per vim mode) block.

**Comment by Stvad** [comment]
Actually none of the standard Roam edit commands would work in Vim "normal mode". Which is at least for me expected, as for me those are "insert mode" commands

**Comment by gvteja** [comment]
That's true. It's probably that todo is something I use very frequently, and so I noticed it in that context first. Would it make sense to have a normal mode shortcut to mimic the cmd + enter functionality?
